business/checkers/authorization_policies_checker.go

```diff
@@ -16,7 +16,7 @@ type AuthorizationPolicyChecker struct {
 	AuthorizationPolicies   []security_v1beta.AuthorizationPolicy
 	Namespace               string
 	Namespaces              models.Namespaces
-	ExportedServiceEntries  []networking_v1alpha3.ServiceEntry
```

Comment:
Note that this checker has "ExportedVirtualServices" as well.
![image](https://user-images.githubusercontent.com/1662329/149790018-45acc8ea-43b1-44a4-975f-d894f966d3a5.png)

It's not used, and I guess it can also be synced.